### PR TITLE
Add support for GitHub actions

### PR DESCRIPTION
**Pull request by Smolevich**
- Add ci file for Github Actions
- Fix work with coveralls.io (remove satooshi/php-coveralls beacuse this package is abandoned and no longer maintained. The author suggests using the php-coveralls/php-coveralls package instead.)

### CODE REVIEW

**Comment by Smolevich** [comment]
@jenssegers, can you review please?

**Comment by Smolevich** [comment]
@jenssegers , can you review please?

**Comment by Smolevich** [comment]
@jenssegers , can you review please?

**Comment by Smolevich** [comment]
@jenssegers [example](https://coveralls.io/github/Smolevich/laravel-mongodb) of coveralls report

**Comment by Smolevich** [comment]
@jenssegers can you review please?

**Comment by Smolevich** [comment]
@jenssegers, can you review please?

**Comment by Smolevich** [comment]
https://github.com/php-coveralls/php-coveralls/pull/275 pr with changes for support GitHub Actions

File: .github/workflows/build-ci.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    branches:
+    tags:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: [7.1, 7.2, 7.3]
```
[the remark this answers is not in the document]
Comment (Smolevich) [reply .github/workflows/build-ci.yml]: We cann add 7.4 to matrix in any time

File: .github/workflows/build-ci.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      run: |
+        composer install --no-interaction
+    - name: Generating code coverage
+      run: |
+        mkdir -p build/logs
+        ./vendor/bin/phpunit --coverage-clover build/logs/clover.xml
+      env:
+        MONGO_HOST: 0.0.0.0
+        MYSQL_HOST: 0.0.0.0
+        MYSQL_PORT: 3307
```
[the remark this answers is not in the document]
Comment (Smolevich) [reply .github/workflows/build-ci.yml]: env variable `MYSQL_PORT` override value in phpunit.xml

File: tests/config/database.php [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 $mongoHost = env('MONGO_HOST', 'mongodb');
 $mongoPort = env('MONGO_PORT') ? (int) env('MONGO_PORT') : 27017;
+$mysqlPort = env('MYSQL_PORT') ? (int) env('MYSQL_PORT') : 3306;
```
[the remark this answers is not in the document]
Comment (Smolevich) [reply tests/config/database.php]: or can to connect or connect was failed

**Comment by jenssegers** [comment]
How do I check if this is working? :)

**Comment by jenssegers** [comment]
I saw that it ran fine on your fork 👍

**Comment by Smolevich** [comment]
Failed build with this problem https://github.com/php-coveralls/php-coveralls/pull/275

**Pull request merged**